src/cascadia/TerminalSettingsEditor/CommonResources.xaml

```diff
@@ -101,22 +94,23 @@
     </Style>
 
     <Style x:Key="AccentBrowseButtonStyle" TargetType="Button" BasedOn="{StaticResource AccentButtonStyle}">
-        <Setter Property="FontSize" Value="{StaticResource StandardFontSize}"/>
         <Setter Property="ToolTipService.Placement" Value="Mouse"/>
         <Setter Property="Margin" Value="10,0,0,0"/>
         <Setter Property="VerticalAlignment" Value="Bottom"/>
     </Style>
 
+    <Style x:Key="IconButtonTextBlockStyle" TargetType="TextBlock">
+        <Setter Property="Margin" Value="10,0,0,0"/>
+    </Style>

```

Comment:
Can't you just use the `BrowseButtonStyle`? If you don't like the name, maybe rename the style too.